Course_Part_04/src/Assignment18.java

```diff
@@ -6,7 +6,6 @@
     public static void main(String[] args) {
         double[] num = {10.5, 56.6,78.0,2.5};
         double sumOfNumbers = averageOfNumbers (num);
-
         System.out.println ("The average of all numbers in the array is " + sumOfNumbers);
     }
 
```

Comment:
de uitwerking van `averageofNumbers()` is prima maar kan eenvoudiger / compacter, ter leer en vermaak:

```
 private static double average(int[] numbers) {
        int sum = 0;
        for (int number : numbers) {
           sum += number;
        }
        return (double) sum / numbers.length;
    }
```